.codespellignore

```diff
@@ -0,0 +1,4 @@
+fo
+raison
+opne
+optimyze
```

Comment:
It seems this name is required for an S3 bucket? Should we plan moving that bucket into the otel org?

Reply:
Makes sense to me, but should not be mixed with this PR.